util/src/main/java/io/zeebe/util/sched/ActorExecutor.java

```diff
@@ -70,6 +71,7 @@ public void start() {
   }
 
   public CompletableFuture<Void> closeAsync() {
+    Loggers.ACTOR_LOGGER.debug("Closing blocking task runner");
```

Comment:
This is not the blocking task runner right?

Reply:
Not? It is closing the task runner afterward `blockingTasksRunner.shutdown()`.

Reply:
This was confusing since it is happening in the ActorExeuctor :sweat_smile: